slotfills/sections/fieldRouter.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+
+import TextField from './text-field';
+import TextareaField from './textarea-field';
+import RichtextField from './richtext-field';
+
+const FieldRouter = ({
+  field,
+  field: {
+    field_class: fieldClass,
+    attributes: {
+      rows = null,
+    } = {},
+    label = '',
+  },
+  index,
+  valueHook,
+}) => {
+  const fieldKey = uuidv4();
```

Comment:
This isn't necessary - keys are only needed when looping. The key should be removed here and in all components below.